src/item/ChorusFruit.php

```diff
@@ -49,22 +49,24 @@ public function onConsume(Living $consumer) : void{
 
 		$origin = $consumer->getPosition();
 		$minX = $origin->getFloorX() - 8;
-		$minY = min($origin->getFloorY(), $consumer->getWorld()->getMaxY()) - 8;
+		$minY = min($origin->getFloorY(), $world->getMaxY()) - 8;
```

Comment:
Unrelated change

Reply:
I know but I think opening a pull request just for that is kinda pointless

Reply:
Changing it is also kind of pointless, so don't make your PR a pain in the ass by adding unrelated changes to it